oci/container.go

```diff
@@ -1,32 +1,37 @@
 package oci
 
 import (
+	"encoding/json"
 	"fmt"
+	"os"
+	"path/filepath"
 	"sync"
 	"time"
 
 	"github.com/containernetworking/cni/pkg/ns"
+	"github.com/docker/docker/pkg/ioutils"
 	specs "github.com/opencontainers/runtime-spec/specs-go"
 	"k8s.io/apimachinery/pkg/fields"
 	pb "k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
 )
 
 // Container represents a runtime container.
 type Container struct {
-	id          string
-	name        string
-	bundlePath  string
-	logPath     string
-	labels      fields.Set
-	annotations fields.Set
-	image       *pb.ImageSpec
-	sandbox     string
+	Id          string
+	Name        string
+	BundlePath  string
+	LogPath     string
+	Labels      fields.Set
+	Annotations fields.Set
+	Image       *pb.ImageSpec
+	Sandbox     string
 	netns       ns.NetNS
-	terminal    bool
-	privileged  bool
-	state       *ContainerState
-	metadata    *pb.ContainerMetadata
+	Terminal    bool
+	Privileged  bool
+	State       *ContainerState
+	Metadata    *pb.ContainerMetadata
 	opLock      sync.Mutex
+	StateDir    string
```

Comment:
Just curious why you didn't upper case netns and oplock too.  Ditto netns in the new container below.